source/module_hamilt_pw/hamilt_pwdft/kernels/cuda/stress_op.cu

```diff
@@ -578,6 +621,10 @@ void cal_stress_drhoc_aux_op<FPTYPE, base_device::DEVICE_GPU>::operator()(
         cal_stress_drhoc_aux2<FPTYPE><<<block,THREADS_PER_BLOCK>>>(
             r,rhoc,gx_arr,rab,drhocg,mesh,igl0,ngg,omega
         );        
+    } else if(type == 3 ){
```

Comment:
Could you write some explanations to explain why there are aux2 and aux3 functions? Otherwise, it will be very difficult for other developers to understand your purposes.

Reply:
Comments have been added to stress_op.h and stress_op.cu.